package.json

```diff
@@ -69,16 +69,18 @@
 	"dependencies": {
 		"@babel/runtime": "^7.15.4",
 		"@react-icons/all-files": "^4.1.0",
+		"bootstrap-icons": "^1.8.0",
```

Comment:
the project have already grommet icons installed can u use it instead

Reply:
Oh yes, I agree with you, but I couldn't find the same design icon in grommet icons, so used bootstrap icon. Do you think it's OK?